Updating the CRISPR section in diagnostics (#589)
greenelab/covid19-review: content/10.diagnostics.md

##### CRISPR-based detection

Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection. 
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.

The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations [@doi:10.1126/science.aam9321].
The target RNA is amplified by RT-RPA and T7 transcription, and the amplified product activates Cas13a. 
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA. 

> An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs [doi:10.1101/2020.02.22.20025460]. 

Author: Thanks for spotting the missing '@'. I was going to flag it and say it might be better to fix it on your end rather than creating another commit and pull request.

It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and 9 ruled out by metagenomic next-generation sequencing, 17 nCoV-/hCoV+ cases and 36 samples from healthy subjects), and a reaction turn-around time of 40 minutes. 
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.

> They determined the limit of detection to be 10 copies/µl using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].

Reviewer: ```suggestion
They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].
```

In their assay, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions [@doi:10.1038/s41587-020-0513-4].
In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the E and N genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay.

> The estimated limit of detection was 10 copies per μl reaction, versus 1 copy per μl reaction for the CDC assay.

Reviewer: ```suggestion
The estimated limit of detection was 10 copies per &mu;l reaction, versus 1 copy per &mu;l reaction for the CDC assay.
```

These results have been confirmed by other DETECTR approaches.

> Using RTRPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per μl of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].

Reviewer: ```suggestion
Using RTRPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per &mu;l of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
```

The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual crRNAs (which increases sensitivity).

> This AIODCRISPR assay (All-In-One Dual CRISPR-Cas12a) detected 4.6 copies of SARS-CoV-2 RNA per μl of input in 40 minutes [@doi:10.1101/2020.03.19.998724].

Reviewer: ```suggestion
This AIODCRISPR assay (All-In-One Dual CRISPR-Cas12a) detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1101/2020.03.19.998724].
```

> This AIODCRISPR assay (All-In-One Dual CRISPR-Cas12a) detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1101/2020.03.19.998724].

Author: Yes, this is correct. I wonder if it could slightly be rephrased too. DETECTR and SHERLOCK are widely used techniques, so the names will make sense to people, but AIOD-CRISPR is a new method.
Maybe something like: "This new All-in-One Dual CRISPR-Cas12a (AIOD-CRISPR) assay detected [...]"

> Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/μl in 40-60 minutes [@doi:10.1101/2020.04.10.023358].

Reviewer: ```suggestion
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1101/2020.04.10.023358].
```
